UACloudLibraryServer/appsettings.json

```diff
@@ -6,5 +6,18 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "DataPlaneSdk": {
+    "ControlApi": {
+      "BaseUrl": "http://localhost:8083/api/control"
+    },
+    "InstanceId": "test-dataplane-instance",
+    "RuntimeId": "example-lock-id",
+    "AllowedSourceTypes": [
+      "test-source-type"

```

Comment:
source and transfer types are particularly important, because they determine which transfers this dataplane can handle. 
The best course of action would be to define (and document!!) a new transfer type.

Reply:
Fixed.